src/frontend/src/eth/stores/eth-transactions.store.ts

```diff
@@ -3,19 +3,19 @@ import type { Transaction } from '$lib/types/transaction';
 import { nonNullish } from '@dfinity/utils';
 import { writable, type Readable } from 'svelte/store';
 
-export type TransactionsData = Record<TokenId, Transaction[]>;
+export type EthTransactionsData = Record<TokenId, Transaction[]>;
 
-export interface TransactionsStore extends Readable<TransactionsData> {
+export interface EthTransactionsStore extends Readable<EthTransactionsData> {
 	set: (params: { tokenId: TokenId; transactions: Transaction[] }) => void;
 	add: (params: { tokenId: TokenId; transactions: Transaction[] }) => void;
 	update: (params: { tokenId: TokenId; transaction: Transaction }) => void;
 	reset: () => void;
 }
 
-const initTransactionsStore = (): TransactionsStore => {
-	const INITIAL: TransactionsData = {} as Record<TokenId, Transaction[]>;
+const initTransactionsStore = (): EthTransactionsStore => {
```

Comment:
You gonna rename `initTransactionsStore` in another PR?

Reply:
sorry, forgot, done in this one